.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 15.x, 16.x, 17.x, 18.x, 20.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Did we also want to bump `engines` in `package.json` to match?  (Currently set to `>= 14.18`)

Reply:
I think we are ok to leave as-is for a little longer. 14.18 is the minimum supported version (due to usage of async/await and other "new ish" bits and pieces). We'll no longer test/support 14.8 because it's reached EOL, but … we don't need to enforce it yet.